CODE_OF_CONDUCT.md

```diff
@@ -33,22 +36,32 @@ Les membres de la Core Team Duchess sont à votre disposition,
 la liste est mise à jour sur [La page Team Duchess](https://www.duchess-france.org/la-team-duchess-france/).
 
 Certaines membres de la Core Team affichent un bandeau sur leur photo dans Slack pour que vous puissiez les reconnaître,
-et sont présentes sur Slack, canal **#moderation**.
+et sont présentes sur Slack, canal **#ask-the-core-team**.
 
 En direct:
 - Pour les évènements en personnes, les organisatrices sur place
-- Angélique sur [twitter](https://twitter.com/aHenryJard) ou Slack #moderation
+- Angélique sur [twitter](https://twitter.com/aHenryJard) ou Slack
 - Florence sur [LinkedIn](https://www.linkedin.com/in/florencechabanois/)
 - Zineb sur [twitter](https://twitter.com/ZinebBendhiba)
 
 ## Sanctions possibles
 
 Si vous ne vous sentez pas en sécurité d’une quelconque façon, vous pouvez contacter une personne de la Core Team Duchess. 
 
-Toute personne qui nuit au bon fonctionnement de la communauté s'expose à des sanctions allant du rappel à l'ordre jusqu'à l'exclusion de la communauté.
+Toute personne qui nuit au bon fonctionnement de la communauté s'expose à des sanctions allant du rappel à l'ordre jusqu'à l'exclusion du Slack et des évènements de la communauté.
 Les sanctions sont à la discrétion de la Core Team Duchess.
 
-## Consignes spécifiques au Slack Duchess
+## Consignes spécifiques au Slack privé Duchess
+
+### Invitation et accès au Slack privé en mixité choisie
+
+Le Slack est **privé** est en mixité choisie, réservé au personnes ayant ces deux critères:
+- femmes ou personne non binaires. Sachant que les femmes cisgenre et les femmes transgenre sont des femmes, et que la transphobie n'est pas tolérée.
+- travaillant dans la tech, ou bien en cours de formation ou reconversion vers un métier de la tech.
+
+L'invitation se fait en contactant la Core Team avec l'email Duchess 'duchessfr [at] gmail.com'. De préférence merci de donner un peu de contexte comme préciser votre poste actuel, et si possible un lien vers un CV ou blog (par exemple LinkedIn).
```

Comment:
Peut être avec une mention de demande possible depuis #ask-the-core-team si on est déja présente sur le slack ?
Et petite reformulation:
`Il est préférable de donner un peu de contexte comme préciser votre poste actuel`